config_dev.toml.example

```diff
@@ -0,0 +1,139 @@
+# Linkedevents environment configuration
+# This file defines a set of (environment) variables that configure most
+# of the functionality of linkedevents. In order for linkedevents to read
+# this file, rename it to `config_dev.toml`. As the name implies, this
+# file is supposed to be used only in development. For production use
+# we recommend setting the environment variables using the facilities
+# of your runtime environment.
+# Linkedevents reads this file by itself. However, it can also be
+# used in conjunction with the included docker-compose.yml. Then
+# you don't need to inject the file into the container. Instead
+# Docker defines environment variables that linkedevents will read.
+# Note that although this file has the extension .toml, it is actually
+# read by a parser only capable of reading key=value pairs + comments.
+# We use the .toml extension to indicate that it is readable by TOML
+# parser.
+# Following are the settings and their explanations, with useful values
+# for development:
+
+# Whether to run Django in debug mode
+# Django setting: DEBUG https://docs.djangoproject.com/en/2.2/ref/settings/#debug
+DEBUG=True
+
+# Namespace ID used for events added through the API. All events added
+# through api will have their "data_source" field set to this value and
+# their identifier will be prefixed by this value.
+SYSTEM_DATA_SOURCE_ID=ledev
```

Comment:
Why is this different from that specified in `settings.py`?

Reply:
They are supposed to be examples for developer. Hmmh. I suppose I'll comment out the settings that would cause harm.

Reply:
Yeah; though an example like `helsinki` (as opposed to `system` which is in `settings.py`) might be a sensible namespace example?

Reply:
Probably. Maybe some other city though. Let's go for "jyvaskyla"? Maybe the namespaces should be explained in the main README.